app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -379,6 +379,16 @@ fun openLink(url: String, ctx: Context, useCustomTab: Boolean) {
     if (useCustomTab) {
         val intent = CustomTabsIntent.Builder()
             .build()
+
+        val url = when {
+            url.startsWith("https://") -> url
+            url.startsWith("/c/") && url.count({ c -> c == '@' }) == 1 -> {
+                val (community, host) = url.split("@", limit = 2)
+                "https://$host$community"
+            }
+            else -> return
```

Comment:
Externalize this to a function, and don't use when. Use if else, or ?? url

Reply:
Since this list will probably grow (I'm guessing `!community@instance` and `/u/user@instance`/`@user@instance`), is there a way you'd prefer to handle this? I could switch it to regex instead if that's preferable.

Reply:
Ok, fixed. The new code is pretty repetitive, but hopefully it's easy to follow. Maybe I'll get around to cleaning it up a bit if I can think of something clever.